python/tests/test_ephemeris.py

```diff
@@ -44,6 +46,12 @@ def test_sat_state():
                                   'af1': 3.637978807091713e-12,
                                   'af2': 0.0}})
   assert eph.good(g.GpsTime(**{ 'wn': 1867, 'tow': 518400.0,}))
+  eph.healthy = 0
```

Comment:
Ack! Figured out what was going on here. 

checkout https://github.com/swift-nav/libswiftnav/blob/tidy-gps-nav/python/swiftnav/ephemeris.pyx#L28.

You'll need to add `fit_interval` to be initialized. Currently, it gets set to the initialized value, which is 0, so this test fails.

Lastly, you can't set properties for the Ephemeris object, since the property setter isn't defined. Consequently, this next line at 49 will fail. You could inline another Ephemeris object with `healthy` set to 0, or you can define property setters for those fields.